python/tk_multi_demo/demos/custom_field_widget/resources/build_resources.sh

```diff
@@ -30,7 +30,7 @@ function build_ui {
 }
 
 function build_res {
-    build_qt "pyside-rcc" "$1.qrc" "$1_rc"
+    build_qt "pyside-rcc -py3" "$1.qrc" "$1_rc"
```

Comment:
This will be a recurring theme in this PR, as quite a few build_resources.sh will be present. They are the bulk of this PR in fact.

Reply:
:+1: